beets/ui/commands.py

```diff
@@ -444,18 +450,25 @@ def show_item_change(item, match):
     """
     cur_artist, new_artist = item.artist, match.info.artist
     cur_title, new_title = item.title, match.info.title
+    cur_album = item.album if item.album else ""
+    new_album = match.info.album if match.info.album else ""
 
-    if cur_artist != new_artist or cur_title != new_title:
+    if (cur_artist != new_artist or cur_title != new_title
+            or cur_album != new_album):
         cur_artist, new_artist = ui.colordiff(cur_artist, new_artist)
         cur_title, new_title = ui.colordiff(cur_title, new_title)
+        cur_album, new_album = ui.colordiff(cur_album, new_album)
 
         print_("Correcting track tags from:")
         print_(f"    {cur_artist} - {cur_title}")
+        print_(f"    Album: {cur_album}") if cur_album else None
```

Comment:
```suggestion
        if cur_album:
            print_(f"    Album: {cur_album}")
```

I don't think I've ever seen this idiom in Python code and it took me a second to parse properly — let's avoid this.

Reply:
Thanks! Well, trying to stay brief/one-line is not _always_ the best idea. I'll also change the other occurence of it.